Planit/Model/SelectedButton.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SelectedButton.swift
+//  Planit
+//
+//  Created by Shivangi on 05/12/22.
+//
+
+import Foundation
+
+enum SelectedButton: String{
+    
+    case all = "All"
+    case Itenary = "Itenary"
+    case packing = "Packing"
+    case travel = "Travel"
+    case tickets = "Tickets"
+    case personal = "Personal"
+    case extra = "Extra"
+    
+    func getValue() -> String{
```

Comment:
I personally don't like variables and functions that use unnecessary verbs like `get`, so I might try to find a nicer name for this. You also don't need this to be a function, because Swift has calculated vars, and that's more appropriate when you don't need to pass arguments, so I would make this something like:

```Swift
var name: String {
    rawValue
}
```